_schemaV2.graphql

```diff
@@ -10225,13 +10226,23 @@ type EditionSet implements Sellable {
   internalID: ID!
   isAcquireable: Boolean
   isForSale: Boolean
+
+  # Is this artwork part of an auction?
+  isInAuction: Boolean
+
+  # Do we want to encourage inquiries on this work?
```

Comment:
#suggestion - `Can collectors submit inquiries on this work/object' ?

Reply:
Ah @oxaudo - i just copied and pasted this stuff right out of the artwork page. Didn't write any of those comments and are historical, so didn't want to change anything.